.eslintrc.yml

```diff
@@ -23,9 +25,23 @@ rules:
   eqeqeq: off
   no-eq-null: off
   no-negated-condition: off
-  no-unused-vars: off
+  no-unused-vars:
+    - error
+    - args: after-used
+      argsIgnorePattern: "^_"
+  no-var: error
   import/no-extraneous-dependencies: error
+  capitalized-comments: off
   prettier/prettier: error
+  no-console: off
+  react/prop-types: off
+  react/display-name: off
+  react/react-in-jsx-scope: off
```

Comment:
I overlooked the matter of `react/react-in-jsx-scope`. Thanks for the advice!

Reply:
When `react/displayName` is turned on, 11 errors were found.
Since it was necessary to designate the function name because it was a lot of change amount, I turned it off this time.

For exapmle......

```diff
- export default () => (
-  <MDXProvider components={components}>
-    <Doc />
-  </MDXProvider>
- )
+ export default function awesomeName() {
+   return (
+     <MDXProvider components={components}>
+       <Doc />
+     </MDXProvider>
+   )
+ }
```

If `react/prop-types` is turned on, 8 errors were found.
Also in this case too much change amount, and because we need to install prop-types, I did not include it in this PR as it is not from the main theme.

I do not normally use prop-types, so I think that I can not throw the PR immediately.

Reply:
It may be better to treat these two as TODO or fix it with another PR.
Like `no-var` this time.

Reply:
That makes sense, thanks for explaining 👍